integrasjon/unleash-klient/src/main/java/no/nav/vedtak/felles/integrasjon/unleash/ToggleConfig.java

```diff
@@ -21,7 +21,7 @@
 
     public static final String PROD = "default";
     public static final String UNLEASH_API_OVERRIDE_KEY = "UNLEASH_API_OVERRIDE";
-    private static final String DEFAULT_UNLEASH_API = "https://unleash.nais.adeo.no/api/";
+    private static final String DEFAULT_UNLEASH_API = "https://unleash.nais.io/api/";
```

Comment:
Eller bruke service discovery?
http://unleash.default.svc.nais.local

Reply:
Gå for SD - så får vi se hvordan det går hvis man bytter cluster